docs/benchmarks/rabi_oscillations.ipynb

```diff
@@ -0,0 +1,488 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        !pip install --quiet cirq
</pre><p>do we need to install recirq in the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/ReCirq/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>